packages/venia-concept/src/components/Footer/footer.js

```diff
@@ -1,9 +1,38 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import classify from 'src/classify';
+import FooterTile from './footerTile';
+import { footerTilePropType } from './constants';
 import defaultClasses from './footer.css';
 
+const footerBlocks = [
+    {
+        iconName: 'user',
+        //TODO: find out link to My Account page
+        headerTitle: () => <Link to="/">Your Account</Link>,
```

Comment:
Here you can just write 

```
{
  headerTitle: <Link to="/">Your Account</Link>
}
```

In this case headerTitle will have prop type node.